application-diagram-ui/src/main/resources/Diagram/DiagramConfig.xml

```diff
@@ -318,9 +318,6 @@
     <property>
       <configurationClass/>
     </property>
-    <property>
-      <configureGlobally>1</configureGlobally>
```

Comment:
If this property was deleted, wasn't it replaced by something else?

Reply:
Hi! I tried exporting the page again, but the property was not replaced with anything.

Reply:
The property is already present:
https://github.com/xwikisas/application-diagram/blob/462f79ac0e95183abf63801f5c99567b09ddfe5c/application-diagram-ui/src/main/resources/Diagram/DiagramConfig.xml#L343